bulletforcehax2_server/src/routing/web_assets.rs

```diff
@@ -3,7 +3,7 @@ use axum::{
     http::{header, Request, Response, StatusCode},
 };
 
-static DIST_DIR: include_dir::Dir = include_dir::include_dir!("$CARGO_MANIFEST_DIR/../dist");
+static DIST_DIR: include_dir::Dir = include_dir::include_dir!("$CARGO_MANIFEST_DIR/assets");
```

Comment:
This is wrong. `dist` is the output directory for building the front-end project, it works as intended. Build the front-end project before attempting to build the back-end without using the live asset server.